readthedocs/docsitalia/management/commands/rebuild_projects.py

```diff
@@ -30,12 +30,22 @@ def add_arguments(self, parser):
             '--document', nargs='?', type=str,
             help='A Read the docs document slug'
         )
+        parser.add_argument(
+            '--version', nargs='?', type=str,
+            help='A Read the docs version slug'
+        )
+        parser.add_argument(
+            '--async', action='store_true', default=False,
+            help='Run the rebuild tasks async'
+        )
 
     def handle(self, *args, **options):
         """handle command"""
         versions = Version.objects.all()
         publisher = options['publisher']
+        version = options['version']
         document = options['document']
+        async = options['async']
```

Comment:
Isn't async a keyword in modern python?

Reply:
you bet it!

Reply:
(only from 3.7 on, but I'm going to remove it)